microsoft-edge/devtools-guide-chromium/performance/overview.md

```diff
@@ -0,0 +1,371 @@
+
+To switch to displaying the initial, **Local metrics** view, click the **Go back to the live metrics page** (![The "Go back to the live metrics page" (house) icon](./overview-images/house-icon.png)) button.
+
+To switch to displaying a different opened recorded profile timeline,  click the **Show recent timeline sessions** dropdown, to the right of the **Go back to the live metrics page** (![The "Go back to the live metrics page" (house) icon](./overview-images/house-icon.png)) button.
+
+![The Performance tool displaying a recorded profile](./overview-images/performance-tool-intro-profile.png)<!-- do not reuse -->
+
+
+<!-- ====================================================================== -->
+## Open the Performance tool
+<!-- https://developer.chrome.com/docs/devtools/performance/overview#open_the_performance_panel -->
+
+To open the **Performance** tool: 
+
+1. Go to a webpage, such as the [Exploring the universe](https://microsoftedge.github.io/Demos/exploring-the-universe/) demo, in a new window or tab.
+
+1. Right-click the webpage, and then select **Inspect**.
+
+   DevTools opens.
+
+1. In the **Activity Bar** at top, click the ![Performance icon](./overview-images/performance-icon.png) **Performance** tool.
+
+   If the ![Performance icon](./overview-images/performance-icon.png) **Performance** tool isn't shown in the **Activity Bar**, click the **More tools** (![The More tools icon](./overview-images/more-tools-icon.png)) button and then select the **Performance** tool.
+
+
+<!-- ------------------------------ -->
+#### Using the Command Menu
+<!-- added heading as separator, compared to upstream -->
+
+To open the **Performance** tool by using the **Command Menu**:
+
+1. Go to a webpage, such as the [Exploring the universe](https://microsoftedge.github.io/Demos/exploring-the-universe/) demo, in a new window or tab.
+
+1. Right-click the webpage, and then select **Inspect**.
+
+   DevTools opens.
+
+1. In the upper right of DevTools, click **Customize and control DevTools**, click **Run command**.
+
+   Or, press:
+
+   * macOS: **Command+Shift+P**
+   * Windows, Linux, ChromeOS: **Control+Shift+P**
+
+   The **Command Menu** opens, and reads **Run >Command**.
+
+1. Start typing **performance**, select **Show Performance [panel]**, and then press **Enter**.
+
+   ![Command Menu with 'performance' entered](./overview-images/command-menu-performance.png)
+
+
+<!-- ====================================================================== -->
+## Monitor Core Web Vitals metrics
+<!-- ## Observe Core Web Vitals live  https://developer.chrome.com/docs/devtools/performance/overview#live-metrics -->
+
+The [Exploring the universe](https://microsoftedge.github.io/Demos/exploring-the-universe/) demo page is designed to load and handle interactions slowly on purpose, in order to illustrate how the LCP, CLS, and INP metrics can be used in the **Performance** tool to identify and fix performance issues.
```

Comment:
I don't think we should talk about the demo yet. Can you move this sentence to the **Use the demo page** section instead?

In fact, this sentence could be under this list item:

* Open a webpage, such as the [Exploring the universe](https://microsoftedge.github.io/Demos/exploring-the-universe/) demo, in a new window or tab.

  The Exploring the universe demo page is designed to load and handle interactions slowly on purpose, in order to illustrate how the LCP, CLS, and INP metrics can be used in the **Performance** tool to identify and fix performance issues.

Reply:
![image](https://github.com/user-attachments/assets/75a11f66-ddea-490f-899d-2e83398e8149)